frontend/pluginloader.lua

```diff
@@ -49,6 +58,11 @@ function PluginLoader:loadPlugins()
         else
             logger.err("extra_plugin_paths config only accepts string or table value")
         end
+    else
+        local datadir = DataStorage:getDataDir()
+        if datadir ~= "." then
+            G_reader_settings:saveSetting("extra_plugin_paths", { datadir .. "/plugins/" })
+        end
```

Comment:
I.e., these lines good, the rest looks like feature creep.

Reply:
Therefore *if you must*, put it in developer options.

Reply:
OK, I see.